test/unit/models/users.test.js

```diff
@@ -128,9 +128,9 @@ describe("users", function () {
     afterEach(async function () {
       await cleanDb();
     });
-    it("It should have discordId field", async function () {
+    it("It should fetch users who have archived:false role", async function () {
```

Comment:
why change the test name to this?

Reply:
Now we are checking on the basis of role archived:false hence the change.

Reply:
Then make the test name more generic like it should fetch users based on discord id